### PR TITLE
[gardening] "if (foo == false)" → "if (!foo)"

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

File: stdlib/public/runtime/CygwinPort.cpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ int swift::_swift_dl_iterate_phdr(int (*Callback)(struct dl_phdr_info *info,
   BOOL ret = EnumProcessModules(procHandle, modules.data(),
                                 modules.size() * sizeof(HMODULE), &neededSize);
 
-  if (ret == FALSE) {
```
Comment (gribozavr) [inline stdlib/public/runtime/CygwinPort.cpp]: What's FALSE defined as on Windows?

Comment (practicalswift) [inline stdlib/public/runtime/CygwinPort.cpp]: I naïvely assumed `0` but I guess we should be on the safe side and check with the Windows/Cygwin guru @tinysun212 :-)

@tinysun212, do you have time to review? That would be appreciated! :-)

Comment (tinysun212) [inline stdlib/public/runtime/CygwinPort.cpp]: #define FALSE   0
#define TRUE    1

They are defined at /usr/include/w32api/ntdef.h in Cygwin.

Comment (practicalswift) [inline stdlib/public/runtime/CygwinPort.cpp]: @tinysun212 Thank you! :+1:

**Pull request merged**